scripts/create_fakes.py

```diff
@@ -310,4 +310,6 @@ def main():
 
 if __name__ == '__main__':
     init_app(set_backends=True, routes=False)
+    for i in range(0,35):
```

Comment:
This should be added to the `main()` function and parametrized rather than hardcoding here.